.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 
 defaults: &defaults
   docker:
-    - image: decidim/decidim:latest-dev
+    - image: decidim/decidim:0.7.2-dev
```

Comment:
Shouldn't we use `0.7.3` here? 🤔

Reply:
We don't have that released. We can use whatever image we want - it's just gonna be more efficient if we have the actual dependencies preloaded in the image.